src/components/maincontroller.ts

```diff
@@ -769,7 +789,7 @@ if (!PRODUCTION) {
 
     window.mediaManager.addEventListener(
         cast.framework.events.category.CORE,
-        (event) => {
+        (event: any) => {
```

Comment:
```suggestion
        (event: Event) => {
```

Reply:
I think there's a type in cast.framework that is more specific here

Reply:
`(event: cast.framework.events.MediaElementEvent)`

Reply:
Hm actually you're right, the closest match seems to be Event.

Reply:
Hm. cast.framework.events.Event